pages/administratorDashboard/main/available_assessments.sql

```diff
@@ -0,0 +1,22 @@
+SELECT c.short_name,ci.long_name,a.title,
```

Comment:
```suggestion
WITH config_for_timezone AS (
    SELECT * WHERE key = 'display_timezone'
)
SELECT c.short_name,ci.long_name,a.title,
```

Reply:
I fixed this by using a sproc, but I appreciate the pointer!